packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -124,6 +204,8 @@ class ListTile extends StatelessWidget {
   final bool isThreeLine;
 
   /// Whether this list tile is part of a vertically dense list.
+  ///
+  /// This property inherits its value from the [ListTileTheme].
```

Comment:
maybe prefer "The default value of this property is based on the [ListTileTheme]", to avoid the word "inherits"? I'm worried about confusing people by being loose with that word given the strict meaning it has in the context of Inherited widgets, which are in play here (I mean, it's not technically the property that is inheriting from the ListTileTheme).